contentcuration/contentcuration/templates/base.html

```diff
@@ -11,6 +11,8 @@
 <html dir="{{ LANGUAGE_BIDI|yesno:'rtl,ltr' }}" lang="{{ LANGUAGE_CODE }}">
     <head>
         {% block head %}
+        <link href="https://fonts.googleapis.com/css2?family=Noto+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
```

Comment:
In [previous work](https://github.com/learningequality/kolibri-server/blob/master/nginx_error_page/error.html#L330), I specified all possibly subsets to support as wide a range of scripts as possible.

https://developers.google.com/fonts/docs/getting_started#specifying_script_subsets

Modern browsers support unicode-ranges, so the extra downloads will only affect older browsers.

Aside: it looks like we are loading italic font variants. As far as I know, no UI elements in the Kolibri Design System use italic, and Kolibri itself does not bundle italic variants. That said, I'm now realizing that this doesn't mean _content_ doesn't use italics, and indeed our rich text exercise editors and HTML apps both support italics. Recommend we investigate and address this...

Reply:
Eventually, it might be nice to add this to the static/fonts folder so that developers can also see what the UI will look like even if their internet gets disconnected

Reply:
It would be much easier for devs to just install Noto to their local machine: https://www.google.com/get/noto/

Including font files locally requires LFS for large binary files, and some fairly complex CSS code to handle all the unicode ranges for different scripts like we do in Kolibri.